openvpn/process_factory.go

```diff
@@ -25,7 +25,11 @@ import (
 )
 
 // CreateNewProcess creates new openvpn process with given config params
-func CreateNewProcess(openvpnBinary string, config *config.GenericConfig, middlewares ...management.Middleware) Process {
+func CreateNewProcess(
```

Comment:
It feels like one-line function declaration was more readable for me.

Reply:
it is better shorter line to comply with 80 symbols length style.

Reply:
I would just call it 'NewProcess', Create and New its two of the same.

Reply:
I think 80 symbols lines is not a limit anymore:
Here is a mention about it in the [Effective Go](https://golang.org/doc/effective_go.html):
> Line length
> Go has no line length limit. Don't worry about overflowing a punched card.

Reply:
But humans do have line length limit, it's all about readability I guess

Reply:
Yep, that is why I mentioned that single line is looking more readable for me.

And another link to the line length in golang: https://github.com/golang/go/wiki/CodeReviewComments#line-length